src/ai/backend/common/docker.py

```diff
@@ -312,6 +312,10 @@ def __init__(self, tags: Iterable[str], value: Optional[str] = None) -> None:
         self._data = dict()
         rx = type(self)._rx_ver
         for tag in tags:
+            if re.match(r"^customized_[0-9a-fA-F]{32}$", tag):
+                self._data["bai_customized_image_tag"] = tag
```

Comment:
Shouldn't you remove `customized_` from `tag`?

Reply:
In this PR, I tried to map the tag itself to the special keyword `bai_customized_image_tag`.
If we’re only mapping the UUID of the custom image, it might be better to rename the keyword to something like `bai_customized_image_id`.